### PR TITLE
Fixed some element changes

### PR DESCRIPTION
**Pull request by psav**
{{pytest: cfme/tests/infrastructure/test_providers.py -k test_provider_crud}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3040146579e78bced033990bbdb1bd617cb2fb5f:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/infrastructure/provider.py [hunk 1]
```diff
@@ -108,7 +113,10 @@ class Provider(Pretty, CloudInfraProvider):
     templates_page_name = "infra_vm_and_templates"
     quad_name = "infra_prov"
     properties_form = properties_form
-    add_provider_button = form_buttons.FormButton("Add this Infrastructure Provider")
+    add_provider_button = deferred_verpick({
+        version.LOWEST: form_buttons.FormButton("Add this Infrastructure Provider"),
+        '5.6': form_buttons.FormButton("Add")
```
Comment (jkrocil) [inline cfme/infrastructure/provider.py]: https://github.com/RedHatQE/cfme_tests/blob/master/cfme/web_ui/form_buttons.py#L122

**Comment by jkrocil** [comment]
Otherwise good :+1:

**Comment by dajoRH** [comment]
Lint report for commit 3e6d9f278a530a39f603a2a5696e8eef81251bc7:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**